src/lib/mongoBackend/mongoUpdateSubscription.cpp

```diff
@@ -888,7 +888,11 @@ std::string mongoUpdateSubscription
 
   if (!noCache)
   {
+    cacheSemTake(__FUNCTION__, "Looking for subscription in cache subscription");
```

Comment:
Perhaps "subscription cache" would have been better wording ...
Too late now, but it could be included in a future PR.